vector/src/main/java/im/vector/app/features/spaces/SpaceExploreActivity.kt

```diff
@@ -49,8 +50,18 @@ class SpaceExploreActivity : VectorBaseActivity<ActivitySimpleBinding>(), SpaceD
 
     val sharedViewModel: SpaceDirectoryViewModel by viewModel()
 
+    private val fragmentLifecycleCallbacks = object : FragmentManager.FragmentLifecycleCallbacks() {
+        override fun onFragmentAttached(fm: FragmentManager, f: Fragment, context: Context) {
```

Comment:
override `onFragmentDetached` to set the listener to `null`?

Reply:
Also `MatrixToBottomSheet.weakReference` could be private

Reply:
done

Reply:
thx